daemon/info.go

```diff
@@ -68,18 +69,26 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		}
 	})
 
-	var securityOptions []string
+	securityOptions := [][2]string{}
```

Comment:
I'll remove the weak info. 

As for the map vs array, we can't return a map from the api because it's not ordered and requires the client to sort it which is awful as you cannot write an app test checking  ordered strings. Let me know what to do.